css-presets/sober.css

```diff
@@ -165,37 +165,48 @@
   --neutral-600: #666666;
   --neutral-800: #262626;
   --neutral-900: #1a1a1a;
+  --secondary-blue: #468ef9;
```

Comment:
Correctly name your variables

Reply:
I have cleaned up the cookie variables that are no more used, one of them was using the `-secondary-blue` variable.
This color is not a configurable variable and not a new introduced one. I renamed it to `--tertiary-color`.

It is used currently only for the plans section (title, price, cta label, cta arrow color when not hovered).

The only configurable variable that by default has this same color is the cookie background color and so we can not use it for plans colors.

* We could introduce new variables but they will be section specific variables.
* Or we can set the `--tertiary-color` as a configurable color that can have this blue color by default and be used for the plans section